oam/install_scripts/mcs-ddlproc.service.in

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=mcs-ddlproc
-PartOf=mcs-writeengineserver.service
+PartOf=mcs-exemgr.service
 After=mcs-dmlproc.service
 
 [Service]
```

Comment:
According with the [doc](https://www.freedesktop.org/software/systemd/man/systemd.service.html) TimeoutStopSec in this and other units w/o ExeStop basically kills the service right away. This is an incorrect thing to do.

Reply:
This is actually not what's actually happening. Systemd will send a TERM signal first, then wait TimeOutSec=*x* seconds (defaults to 90), before sending KILL